spec.bs

```diff
@@ -2664,6 +2691,81 @@ To <dfn>fetch trusted signals</dfn> given a [=URL=] |url|, an [=origin=] |script
   1. Return « |signals|, |perInterestGroupData|, |dataVersion| ».
 </div>
 
+<div algorithm>
+To <dfn>fetch trusted key value signals</dfn> given a [=URL=] |url|, a [=byte sequence=] |body|, a [=string=]
+|context|, an [=origin=] |scriptOrigin|, a [=policy container=] |policyContainer|, an |indexMap|, and a
+[=boolean=] |isBiddingSignal|:
+
+1. Let |request| be a new [=request=] with the following properties:
+  :   [=request/URL=]
+  ::  |url|
+  :   [=request/method=]
+  ::  `POST`
+  :   [=request/body=]
+  ::  |body|
+  :   [=request/origin=]
+  ::  |scriptOrigin|
+  :   [=request/header list=]
+  ::  «`Accept`: `application/json`»
+  :   [=request/client=]
+  ::  `null`
+  :   [=request/mode=]
+  ::  "`cors`"
+  :   [=request/referrer=]
+  :: "`no-referrer`"
+  :   [=request/credentials mode=]
+  ::  "`omit`"
+  :   [=request/redirect mode=]
+  :: "`error`"
+  :   [=request/policy container=]
+  ::  A new [=policy container=] whose [=policy container/IP address space=] is |policyContainer|'s
+    [=policy container/IP address space=]
+
+  Issue: One of the side-effects of a `null` client for this subresource request is it neuters all

```

Comment:
A [pull request](https://github.com/WICG/turtledove/pull/1351/files#diff-6f5a1d8263b0b0c42e2716ba5750e3652e359532647ac934c1c70086ae3ceddaR2599) that updated this just landed. It removes this note and explicitly set a service worker mode. Please update this as well following that.